easybuild/easyconfigs/w/WhatsHap/WhatsHap-1.1-foss-2020b.eb

```diff
@@ -0,0 +1,43 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PythonBundle'
+
+name = 'WhatsHap'
+version = '1.1'
+
+homepage = 'https://whatshap.readthedocs.io'
+description = """WhatsHap is a software for phasing genomic variants using DNA
+sequencing reads, also called read-based phasing or haplotype assembly. It is
+especially suitable for long reads, but works also well with short reads."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('Pysam', '0.16.0.1'),
+    ('networkx', '2.5'),
+    ('SciPy-bundle', '2020.11'),
+    ('Biopython', '1.78'),
+    ('Yasm', '1.3.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('pyfaidx', '0.5.9.5', {
```

Comment:
@arielzn We have a standalone easyconfig for `pyfaidx` already that is compatible with `foss/2020b`, so please make this a proper dependency